README.md

```diff
@@ -143,7 +142,7 @@ $ gm convert girl.jpg -hald-clut halds/15.Hudson.png girl.15.jpg
 You can notice that in the processed image many objects look flat and posterized:
 face, hair, chairs in the background.
 While posterization is one of the common image filters, it is not a part of the
-Hudson filter.
+Hudson-like filter.
```

Comment:
I believe it is not a part of the original Hudson filter neither reconstructed one.

Reply:
Well, we pretty much talked about not calling filters with their original Instagram names, that's way we've got the -like thing around